pkg/util/interrupt/child_linux.go

```diff
@@ -24,4 +24,4 @@ import (
 )
 
 // childSignals are the allowed signals that can be sent to children in Unix variant OS's
-var childSignals = []os.Signal{syscall.SIGINT, syscall.SIGTERM, syscall.SIGQUIT}
```

Comment:
and there are [build constraints](https://golang.org/pkg/go/build/#hdr-Build_Constraints) in the files as well.